deploy/smcp-templates/servicemesh

```diff
@@ -27,6 +27,9 @@ spec:
       image: citadel-rhel8
     sidecarInjectorWebhook:
       image: sidecar-injector-rhel8
+    gateways:
+      istio-ingressgateway:
+        ior_image: istio-ior-rhel8
```

Comment:
I know this is how it used be handled, but is that really something we should put into `istio-ingressgateway`? Maybe we should rethink this while we're at it.

I'd much prefer if we introduced ior as a component just like other Istio components:
```yaml
spec:
  istio:
    ior:
      enabled: false
      image: istio-ior-rhel8```